### PR TITLE
Personal website

### PR DESCRIPTION
**Pull request by RuslanMirasov**

### CODE REVIEW

File: html-and-web/html-and-the-web_personal-website/index.html [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Personal Website</title>
+  </head>
+  <body>
+    <header class="header">
+      <h1>Ruslan Mirasov</h1>
+      <nav class="nav">
+        <a href="#contact">Contact</a>
```
Comment (Nicki-lombardo) [inline html-and-web/html-and-the-web_personal-website/index.html:13]: I'm not sure if the # is necessary. Could you check it!

File: html-and-web/html-and-the-web_personal-website/index.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        <a href="#biography">Biography</a>
```
Comment (Nicki-lombardo) [inline html-and-web/html-and-the-web_personal-website/index.html:14]: I'm not sure if the # is necessary. Could you check it!

File: html-and-web/html-and-the-web_personal-website/index.html (hunk 1 above)
Comment (RuslanMirasov) [reply html-and-web/html-and-the-web_personal-website/index.html:13]: It's fine, this page won't link to other pages, the links should just take you to a specific section on the current page. So it's okay.

File: html-and-web/html-and-the-web_personal-website/index.html (hunk 2 above)
Comment (RuslanMirasov) [reply html-and-web/html-and-the-web_personal-website/index.html:14]: It's fine, this page won't link to other pages, the links should just take you to a specific section on the current page. So it's okay.

File: html-and-web/html-and-the-web_personal-website/index.html (hunk 1 above)
Comment (Nicki-lombardo) [reply html-and-web/html-and-the-web_personal-website/index.html:13]: Well done!

Comment (Nicki-lombardo) [reply html-and-web/html-and-the-web_personal-website/index.html:13]: ok!

**Review by Nicki-lombardo — COMMENTED** [review]
Well done!

**Pull request merged**